packages/dashboard/src/pages/organization-page/organization-page.tsx

```diff
@@ -0,0 +1,135 @@
+import { Component, createEffect, createSignal, Show } from 'solid-js';
+
+import { Button, FormItem, TabItem, Tabs } from '@castmill/ui-common';
+import { OrganizationMembersView } from './organization-members-view';
+import { store } from '../../store';
+import { BsCheckLg } from 'solid-icons/bs';
+
+import style from './organization-page.module.scss';
+import { OrganizationsService } from '../../services/organizations.service';
+import { OrganizationInvitationsView } from './organization-invitations-view';
+
+const OrganizationPage: Component = () => {
+  const params = useSearchParams();
+
+  const [name, setName] = createSignal(store.organizations.selectedName!);
+
+  const [isFormModified, setIsFormModified] = createSignal(false);
+  const [errors, setErrors] = createSignal(new Map());
+
+  const validateField = (fieldId: string, value: string) => {
+    let error = '';
+    switch (fieldId) {
+      case 'name':
+        if (!value) {
+          error = 'Name is required';
+        } else if (value.length < 5) {
+          error = 'Name must be at least 5 characters';
+        }
+        break;
+      default:
+        error = '';
+    }
+
+    setErrors((prev) => new Map(prev).set(fieldId, error));
+    return !error;
+  };
+
+  createEffect(() => {
+    // We need to check both fields or create effect will not detect the dependencies.
+    const hasModifiedName = name() !== store.organizations.selectedName;
+    setIsFormModified(hasModifiedName);
+
+    if (store.organizations.selectedId) {
+      setName(store.organizations.selectedName);
+    }
+  });
+
+  const isFormValid = () => {
+    return ![...errors().values()].some((e) => e) && isFormModified();
+  };
+
+  const onSubmit = async (organization: { id: string; name: string }) => {
+    try {
+      await OrganizationsService.update(organization.id, {
+        name: organization.name,
+      });
+    } catch (error) {
+      alert(`Error updating organization ${error}`);
+    }
+  };
+
+  const resourcesTabs: TabItem[] = [
+    {
+      title: 'Users',
+      content: () => (
+        <Show when={store.organizations.selectedId}>
+          <OrganizationMembersView
+            organizationId={store.organizations.selectedId!}
+            organizationName={store.organizations.selectedName!}
+            onRemove={() => {}}
+          />
+        </Show>
+      ),
+    },
+    {
+      title: 'Invitations',
+      content: () => (
+        <Show when={store.organizations.selectedId}>
+          <OrganizationInvitationsView
+            organizationId={store.organizations.selectedId!}
+            onRemove={() => {}}
+          />
+        </Show>
+      ),
+    },
+  ];
+
+  return (
+    <div class={style['organization-page']}>
+      <div class={style['header']}>
+        <h1>Organization</h1>
+        <form
+          onSubmit={async (e) => {
+            e.preventDefault();
+            if (isFormValid()) {
+              const team = await onSubmit({
```

Comment:
Unused variable. If it is needed it should probably be called `organization` and not `team`